bindist/linux/debian/scripts/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Bootstrap Python installation and run build.py
+set -x -euf -o pipefail
+
+cd $(dirname $0)
+apt update
+apt install python3 -y
```

Comment:
Isn't this run inside the docker container? If so why are you installing python3 here again?

Reply:
I didn't want to make another docker container due to the limits on dockerhub.com.. So the `Dockerfile` caches _some_ things to make it cheaper to experiment locally. But it should all work on base images.